### DISCUSSION TITLE
flinkcdc支持将posgtresql的分区表同步到doris中吗？求大佬们指教

### DISCUSSION THREAD

**Discussion by IT-Forerunner** [apache/flink-cdc#2733] [Q&A]
问题描述：flinkcdc将pg数据的表同步到doris中，正常的表都能同步，就是分区表同步一直读取不到数据。
以下是简单个pg分区表建表语句：
CREATE TABLE "xb_test_tb" (
  "pk_uid" int8 NOT NULL ,
  "id" varchar(50) COLLATE "pg_catalog"."default",
  "sheng" varchar(2) COLLATE "pg_catalog"."default" NOT NULL,
)
PARTITION BY LIST (
  "sheng" COLLATE "pg_catalog"."default" "pg_catalog"."text_ops"
)
;
ALTER TABLE "xb_test_tb" ATTACH PARTITION "xb_test_tb_11_beijing" FOR VALUES IN ('11');
ALTER TABLE "xb_test_tb" ATTACH PARTITION "xb_test_tb_65_xinjiang" FOR VALUES IN ('65');
ALTER TABLE "xb_test_tb" ATTACH PARTITION "xb_test_tb_91_dxalsg" FOR VALUES IN ('91');
ALTER TABLE "xb_test_tb" ATTACH PARTITION "xb_test_tb_92_longjiangsengong" FOR VALUES IN ('92');
ALTER TABLE "xb_test_tb" ATTACH PARTITION "xb_test_tb_93_neimenggusengong" FOR VALUES IN ('93');
ALTER TABLE "xb_test_tb" ATTACH PARTITION "xb_test_tb_94_jilinsengong" FOR VALUES IN ('94');
ALTER TABLE "xb_test_tb" ATTACH PARTITION "xb_test_tb_95_xjbt" FOR VALUES IN ('95');

请教大佬们两个问题。
1. flinkcdc是否支持同步分区表。
2. flinkcdc同步任务sql有什么特殊配置吗。
3. flinkcdc同步pg分区表能当作普通的表同步吗，doris建普通的表。

**Comment by 2606090723** [comment]
我也是读不到 ,你解决了吗
